### PR TITLE
[stdlib] Add examples for `env_get_int` and `sizeof`

### PR DESCRIPTION
**Pull request by rd4com**
Hello, this commit adds a few examples:
- `env_get_int` for parametrizing
- `sizeof` for sizes

### CODE REVIEW

**Comment by rd4com** [comment]
❌ Run pre-commit / lint (pull_request):

```
Have to finish with `.`
```

✅ Had to add a note after one example:

```
Note: `align_of` is in same module.
```

**Comment by JoeLoser** [comment]
> ❌ Run pre-commit / lint (pull_request):
> 
> ```
> Have to finish with `.`
> ```
> 
> ✅ Had to add a note after one example:
> 
> ```
> Note: `align_of` is in same module.
> ```

Do you mind filing a GH issue for this?  I think we should loosen our parsing doc requirements to allow for things of the form

```mojo
some example code here
```

without ending in a `.` at the end after the code snippet.

**Review by JoeLoser — APPROVED** [review]

File: stdlib/src/sys/param_env.mojo [hunk 1]
```diff
@@ -76,6 +76,27 @@ fn env_get_int[name: StringLiteral, default: Int]() -> Int:
...
+
+    fn parametrized[num: Int]():
+        print(num)
+    ```
+
+    If the program is `app.mojo`:
+    - `mojo run -D favorite_number=2 app.mojo`
+    - `mojo run -D app.mojo`
+
+    Note: useful for parametrizing SIMD vector sizes.
```
Comment (JoeLoser) [inline stdlib/src/sys/param_env.mojo:99]:
```suggestion
    Note: useful for parameterizing SIMD vector sizes.
```

**Comment by JoeLoser** [comment]
!sync

**Pull request closed without merging**